src/app/containers/IndexPageSection/utilities/storyRowsSplitter.test.jsx

```diff
@@ -259,13 +259,17 @@ const numberOfStories = {
 
```

Comment:
We should add a couple of tests here to test the behaviour when `showAllPromos` is true